pokete_classes/loops.py

```diff
@@ -10,14 +11,16 @@ def easy_exit_loop():
             _ev.clear()
             return
         std_loop()
-        time.sleep(0.05)
 
 
-def std_loop(on_mvmp=True):
+def std_loop(on_mvmp=True, pevm=None):
     """Standart action executed in most loops
     ARGS:
         on_mvmp: Indicates if the loop is executed on movemap"""
     if _ev.get() == "exit":
         raise KeyboardInterrupt
     if on_mvmp:
         notifier.next()
+    if pevm != None:
```

Comment:
Checking for `None` is better done with `is not None`:

```suggestion
    if pevm is not None:
```